packages/server/src/lib/grantsCollaboration/notes.js

```diff
@@ -100,4 +117,28 @@ async function getOrganizationNotesForGrant(knex, grantId, organizationId, { aft
     };
 }
 
-module.exports = { saveNoteRevision, getOrganizationNotesForGrant };
+async function getOrganizationNotesForGrantByUser(
+    knex,
+    organizationId,
+    userId,
+    grantId,
+    { afterRevision, limit = 50 } = {},
+) {
+    return getCurrentNoteRevisions(knex, { grantId, userId }, { afterRevision, limit });
```

Comment:
Per https://github.com/usdigitalresponse/usdr-gost/issues/3426#issuecomment-2358790420:
```suggestion
    return getCurrentNoteRevisions(knex, { grantId, organizationId, userId }, { afterRevision, limit });
```